config/crd/bases/infrastructure.cluster.x-k8s.io_baremetalclusters.yaml

```diff
@@ -142,6 +142,10 @@ spec:
       description: Cluster to which this BMCluster belongs
       name: Cluster
       type: string
+    - JSONPath: .spec.controlPlaneEndpoint
+      description: API endpoint
+      name: APIEndpoints
```

Comment:
ContolPlane Endpoint may be to be consistent with the field name? I would also add a few more words in description.

Reply:
done